src/main/kotlin/fi/metatavu/metaform/server/rest/AbstractApi.kt

```diff
@@ -151,6 +151,17 @@ abstract class AbstractApi {
         return "$target with ID $id could not be found"
     }
 
+    /**
+     * Creates not found message with given parameters
+     *
+     * @param target target of the find method
+     * @param slug metaform slug
+     * @return not found message
+     */
+    protected fun createNotFoundMessage(target: String, slug: String): String {
```

Comment:
This is not generic enough, to be called createNotFoundMessage, instead it should be called, createSlugNotFoundMessage